ttnn/cpp/ttnn/operations/ccl/reduce_scatter/device/reduce_scatter_op.cpp

```diff
@@ -55,10 +55,13 @@ operation::ProgramWithCallbacks ReduceScatter::create_program(
 }
 
 static ttnn::operations::binary::BinaryOpType convert_reduce_type_to_eltwise_type(ttnn::operations::reduction::ReduceType reduce_op) {
+    // Leaving switch statement for future support of additional types.
     switch (reduce_op) {
-        case ttnn::operations::reduction::ReduceType::Sum: return ttnn::operations::binary::BinaryOpType::ADD;
-
-        default: TT_FATAL("Reduce scatter only support reduce_type Sum"); return ttnn::operations::binary::BinaryOpType::ADD;
+        case ttnn::operations::reduction::ReduceType::Sum:
+            return ttnn::operations::binary::BinaryOpType::ADD;
+        default:
+            tt::log_info(tt::LogOp, "Reduce scatter only supports reduce_type Sum. Ignoring type of {} and acting as if that were passed in.", reduce_op);
```

Comment:
Don't know how I feel about this. At the very least should be log_warning.

Reply:
I'll turn this into a warning. I can also move it back to a fatal if you'd like.

Reply:
Updated.

Reply:
I did think about it and I I would prefer to keep it as a THROW  since this is a user specified reduction type and if they specified a wrong or unsupported one, then we should just fail since add won't be a viable substitute.

Reply:
Updated.